src/CircleEditor/CircleEditorDocument.ts

```diff
@@ -191,4 +195,333 @@ export class CircleEditorDocument extends Disposable implements vscode.CustomDoc
       }
     };
   }
+
+  /**
+   * Guess data's type (int or float)
+   */
+  private guessExactType(n: any) {
+    if (Number(n) % 1 === 0) {
+      return "int";
+    }
+    else if (Number(n) % 1 !== 0) {
+      return "float";
+    }
+  }
+
+  /**
+   * Encode key and value entered in the calculator using flexbuffers.
+   * Send encoded data.
+   */
+  public sendEncodingData(message: any) {
+    let fbb = flexbuffers.builder();
+    fbb.startMap();
+    for (const key in message.data) {
+      fbb.addKey(key);
+      const val = message.data[key][0];
+      const valType = message.data[key][1];
+      if (valType === "boolean") {
+        if (val === "true" || val === true) {
+          fbb.add(true);
+        }
+        else if (val === "false" || val === false) {
+          fbb.add(false);
+        }
+        else {
+          Balloon.error("'boolean' type must be 'true' or 'false'.", false);
+          return;
+        }
+      }
+      else if (valType === "int") {
+        if (this.guessExactType(val) === 'float') {
+          Balloon.error("'int' type doesn't include decimal point.", false);
+          return;
+        }
+        fbb.addInt(Number(val));
+      }
+      else {
+        fbb.add(String(val));
+      }
+    }
+    fbb.end();
+    const res = fbb.finish();
+    const buf = Buffer.alloc(res.byteLength);
+    const view = new Uint8Array(res);
+    for (let i = 0; i < buf.length; ++i) {
+      buf[i] = view[i];
+    }
+    const data = Array.from(buf);
```

Comment:
Just curious, isn't it possible to use following codes?

```suggestion
    const data = Array.from(res);
```

Reply:
Oh!! it is possible!
I appreciate it.